report/sections/sprint2/sprint-2-planning.tex

```diff
@@ -36,7 +36,9 @@ \subsection{The goals and visions for the GIRAF project}
 When a group finishes implementing their user story during the sprint, they should discuss if they had time to implement a new user story before the sprint ended. They should then present their chosen user story to the PO group and get a confirmation.
 
 \subsubsection{The goals for the PO group}
-One of the goals for the PO group in sprint 2 was to implement the user stories assigned to our group.
+We had multiple goals in sprint 2.
+The first was to continue producing and updating prototypes.
+On top of this, we had to implement the user stories assigned to our group.
```

Comment:
Kan man ikke skrive noget i stil med:

Our goals for this sprint was mainly to keep working on prototypes and implement the user stories assigned to our group?

Bare for at have en sætning i stedet for 3 små